Sources_v3/Controllers/ChannelController_Tests.swift

```diff
@@ -275,6 +275,28 @@ class ChannelController_Tests: StressTestCase {
     }
 
     // MARK: - Channel actions propagation tests
+    
+    func test_updateChannel_blocked_by_isChannelCreatedFlag() {
+        //  Create `ChannelController` for new channel
+        let query = ChannelQuery(channelPayload: .unique)
+        controller = ChannelController(channelQuery: query, client: client, environment: env.environment, isChannelCreated: false)
+        controller.startUpdating()
+
+        // Simulate `updateChannel` call
+        controller.updateChannel(team: nil, extraData: .init())
+
+        // Assert method is not called before channel creation
+        XCTAssertNil(env.channelUpdater!.updateChannel_payload?.cid)
+
+        // Simulate succsesfull backend channel creation
+        env.channelUpdater!.update_channelCreatedCallback?(query.cid)
+
+        // Simulate `updateChannel` call
+        controller.updateChannel(team: nil, extraData: .init())
+
+        // Assert method is called after channel creation
+        XCTAssertEqual(env.channelUpdater!.updateChannel_payload?.cid, query.cid)
```

Comment:
Since the function will return an error you can just check the error here. You can use our `await` helper for that.